DiscordChatExporter.Core/Exporting/ExportContext.cs

```diff
@@ -138,4 +138,10 @@ public async ValueTask<string> ResolveAssetUrlAsync(string url, CancellationToke
             return url;
         }
     }
+
+    internal async Task<IEnumerable<User>> ResolveReactionUsers(Snowflake messageId, Emoji emoji, CancellationToken cancellationToken)
+    {
+        var users = await Discord.GetMessageReactionsAsync(Request.Channel.Id, messageId, emoji, cancellationToken);
+        return users;
+    }
```

Comment:
Since this doesn't rely on any state, you could just call `Context.Discord.GetMessageReactionsAsync(...)` instead, and remove this method.